src/newViews/AnalyticalFramework/SecondaryTagging/index.tsx

```diff
@@ -27,20 +39,99 @@ function SecondaryTagging(props: Props) {
 
     // NOTE: intentional console.info
     console.info('secondary tagging in the framework', frameworkId);
```

Comment:
I didn't add this. It's added so that analysisFrameworkId is used anyhow.
Either we should modify the interface or have a console.log.
I am okay with this as it is now.